.github/workflows/e2e-local.yml

```
@@ -0,0 +1,96 @@
+name: E2E tests / Locally
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  e2e-tests:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install common tools
+        working-directory: ./deployments/local-setup-environment/provisioning/
+        run: ./install-commons.sh
+
+      - name: Use Node.js 12
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-solidity-node-modules
+        with:
+          path: ~/.npm # npm cache files are stored in `~/.npm` on Linux/macOS
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+          
+      - name: Install Go
+        working-directory: ./deployments/local-setup-environment/provisioning/
+        run: |
+          sudo rm -rf /usr/local/go
+          ./install-go.sh
+          echo "Using Go: $(go version)"
+
+      - name: Set GOPATH variable
+        run: echo "GOPATH=$HOME/go" >> $GITHUB_ENV
+
+      - name: Append PATH variable
+        run: |
+          echo "/usr/local/go/bin" >> $GITHUB_PATH
+          echo "$GOPATH/bin" >> $GITHUB_PATH
```

Can we merge these three?
```suggestion
      - name: Install Go
        working-directory: ./deployments/local-setup-environment/provisioning/
        run: |
          sudo rm -rf /usr/local/go
          ./install-go.sh
          echo "GOPATH=$HOME/go" >> $GITHUB_ENV
          echo "/usr/local/go/bin" >> $GITHUB_PATH
          echo "$GOPATH/bin" >> $GITHUB_PATH
          echo "Using Go: $(go version)"
```

---

Only partially. 
Adding GOPATH to the list of environment variables available to GITHUB can be performed in the same step in which Go gets installed. However, we cannot include in that step the command which appends the `$GITHUB_PATH` with `$GOPATH/bin` value, as the `$GOPATH` variable would not be available at this stage yet.
I merged the `Set GOPATH variable` step with `Install Go`step, see here: https://github.com/keep-network/local-setup/commit/047700f8a1643a0605fd19670a4e9216a465d286

---

Maybe we could do it in one step without using `GOPATH`?
```
echo "GOPATH=$HOME/go" >> $GITHUB_ENV
echo "$HOME/go/bin" >> $GITHUB_PATH
echo "/usr/local/go/bin" >> $GITHUB_PATH
```